giraffe/src/components/geo/processing/toolTips.ts

```diff
@@ -0,0 +1,39 @@
+// Types
+import {Axis} from '../../../types/geo'
+
+export const formatValue = (
+  key: string,
+  defaultLabel: string,
+  value: number,
+  dimension: Axis = {}
+) => {
+  const {label, prefix = '', suffix = ''} = dimension
+  const formattedValue = `${prefix}${value}${suffix}`
+  if (value !== undefined && value !== null)
```

Comment:
Please always use braces for if statements. We will be updating the eslint rules to reflect this.